lib/code_corps/emails/forgot_password_email.ex

```diff
@@ -0,0 +1,18 @@
+defmodule CodeCorps.Emails.ForgotPasswordEmail do
+  import Bamboo.Email
+  import Bamboo.PostmarkHelper
+
+  alias CodeCorps.{Emails.BaseEmail, AuthToken}
+
+  def create(user, token) do
+    BaseEmail.create
+    |> to(user.email)
+    |> template(template_id(), [link: link(token)])
+  end
+
+  defp template_id, do: Application.get_env(:code_corps, :postmark_forgot_password_template)
+
+  defp link(token) do
+    "#{token}" # TODO: we can replace this with the proper string later
```

Comment:
@JoshSmith That's the general idea. Not sure if we want to go with the config way of doing this (set this route as a config value) or something else.